Artesian/Artesian.SDK/Factory/IMarketData.cs

```diff
@@ -212,10 +212,11 @@ public interface IMarketAssessmentWritable
         /// <param name="downloadedAt">The instant downloaded</param>
         /// <param name="deferCommandExecution">Defer Command Execution</param>
         /// <param name="deferDataGeneration">Defer Data Generation</param>
+        /// <param name="upsertMode">Upsert Mode</param>
         /// <param name="keepNulls">if <see langword="false"/> Settlement=null are ignored (server-side). That is the default behaviour.</param>
         /// <param name="ctk">Cancellation Token</param>
         /// <returns></returns>
-        Task Save(Instant downloadedAt, bool deferCommandExecution = false, bool deferDataGeneration = true, bool keepNulls = false, CancellationToken ctk = default);
+        Task Save(Instant downloadedAt, bool deferCommandExecution = false, bool deferDataGeneration = true, bool keepNulls = false, UpsertMode? upsertMode = null, CancellationToken ctk = default);
```

Comment:
missing the overloads, same as per the classes.

Reply:
@AndreaCuneo Would it be better to have UpsertMode default to Merge rather than be nullable?